docs/application/vstools/tools/emulator-manager.md

```diff
@@ -126,16 +128,17 @@ In the Emulator Manager, you can launch, edit, delete, reset, and export emulato
 3. Manage the instance:
    - To launch the emulator, click **Launch**.
 
-     The debug and launch options are activated after the emulator has booted.
+     The debug and launch options are activated after the emulator has booted. The name of the emulator instance is displayed on the Visual Studio toolbar.
 
 	 ![Launched emulator instance](media/em-launched.png)
 
-	 The name of the emulator instance is displayed on the Visual Studio toolbar.
-
      ![Activated debug option](media/em-activated-debug.png)
 
    - To edit an emulator, click **Edit**, make the desired changes, and click **Confirm**.
    - To delete an emulator, click **Delete**.
    - To reset an emulator, right-click it and select **Reset**.
-   - To export an emulator, right-click it and select **Export As**. Specify the new image file location.  
-   When you export the emulator instance, the state of the platform image is replicated.
+
+<a name="export"></a>
+     To export an emulator, right-click it and select **Export as**. Specify the new image file location.
```

Comment:
Did you pull this item out of the list on purpose? `To edit`, `delete`, `reset` and `export` were in one list in original file.